src/cqueue.rs

```diff
@@ -160,19 +177,23 @@ impl Drop for AvailableQueue<'_> {
 }
 
 impl Entry {
-    /// Result value
+    /// If successful, the number of bytes transferred. If an error occurred, this is a negative
```

Comment:
> If successful, the number of bytes transferred. If an error occurred, this is a negative errno code.

I think the meaning of result depends on opcode.

Reply:
Yes, so for example I'm using `opcode::Openat2` where the result value, when positive, is the opened file descriptor.  BTW: This raises the question of if Entry should be generic over opcode with some associate types or the like.  It would seem better if for, `opcode::Openat2` result was `Result<opcode::types::Fd, io::Error>` but maybe that necessitates some higher level layer?

Reply:
The `io-uring` is low-level abstraction and should not do too much.